--- priv/riak.schema.orig	2021-09-07 16:51:09.000000000 +0300
+++ priv/riak.schema	2021-09-07 16:43:55.000000000 +0300
@@ -74,6 +74,7 @@
       undefined -> [];
       ErrorFilename -> [{lager_file_backend, [{file, ErrorFilename},
                                               {level, error},
+                                              {formatter, lager_logstash_formatter},
                                               {size, 10485760},
                                               {date, "$D0"},
                                               {count, 5}]}]
@@ -85,6 +86,7 @@
     ConsoleHandler = {lager_console_backend, ConsoleLogLevel},
     ConsoleFileHandler = {lager_file_backend, [{file, ConsoleLogFile},
                                                {level, ConsoleLogLevel},
+                                               {formatter, lager_logstash_formatter},
                                                {size, 10485760},
                                                {date, "$D0"},
                                                {count, 5}]},
