--- deps/riak_core/src/riak_core_handoff_status.erl.orig	2020-04-08 14:27:24.000000000 +0300
+++ deps/riak_core/src/riak_core_handoff_status.erl	2020-04-08 14:31:55.000000000 +0300
@@ -133,7 +133,7 @@
 format_transfer_size({Num, objects}) ->
     io_lib:format("~B Objs", [Num]);
 format_transfer_size({Num, bytes}) ->
-    riak_core_format:human_size_fmt("~B", Num);
+    riak_core_format:human_size_fmt("~.2f", Num);
 format_transfer_size(_) ->
     "--".
 
